quelpa.el

```diff
@@ -267,7 +267,7 @@ Return nil if the package is already installed and should not be upgraded."
   "Build a package from the given recipe RCP.
 Uses the `package-build' library to get the source code and build
 an elpa compatible package in `quelpa-build-dir' storing it in
-`quelpa-packages-dir'. Return the path to the created file or nil
+`quelpa-packages-dir'.  Return the path to the created file or nil
```

Comment:
I'm curious, why does it have two space here now? Is that an elisp rule or something?

Reply:
Yes, that is standard Emacs rules, which `checkdoc` warns about.  The simplest reason is that it allows `M-a`/`M-e` to move between sentences, which is very convenient when editing prose.

Reply:
:+1: